### ISSUE TITLE
Clicking on "View on etherscan.io" for NFTs also opens NFT preview

### ISSUE THREAD

**Issue by KevinLu** [safe-global/web-core#1796]
## Bug description
Clicking on "View on etherscan.io" for NFTs also opens NFT preview in the Safe app. When I go back to the Safe app, the NFT preview is opened unexpectedly.

## Environment
 - Browser: Chrome latest
 - Wallet: MetaMask
 - Chain: Ethereum mainnet

## Steps to reproduce
 1. Go to a Safe with some NFTs, for example: https://app.safe.global/balances/nfts?safe=eth:0xf26d1Bb347a59F6C283C53156519cC1B1ABacA51
 2. Click on the "View on etherscan.io" button for the NFT 

## Expected result
Etherscan link opens in new tab only, nothing else should happen.

## Obtained result
Etherscan link opens in new tab and the NFT preview is opened in the Safe app.

## Screenshots
<img width="403" alt="image" src="https://user-images.githubusercontent.com/18564130/227005395-849a582f-9450-4876-889c-77cf6450f00c.png">

**Comment by katspaugh** [comment]
Thanks for the report!
Something tells me you know how to fix it, so please feel free to open a PR :)

**Comment by KevinLu** [comment]
> Thanks for the report! Something tells me you know how to fix it, so please feel free to open a PR :)

Maybe I'm mistaken but I thought this PR fixed it? https://github.com/safe-global/safe-react-components/pull/220
Unless it's a different problem than stopping the event propagation?

**Comment by katspaugh** [comment]
Indeed! Looks like we just need to update that package version here.

**Comment by KevinLu** [comment]
> Indeed! Looks like we just need to update that package version here.

I don't think any PR is required here unless you want to specifically bump up the package version to `2.0.3`. It's already set to `^2.0.2` in the `package.json` so it should use `2.0.3` as it's the latest available.

**Comment by katspaugh** [comment]
@KevinLu I'm fixing something else in that package, so I'll release 2.0.4 soon and update it here.
